app/components/pages/BrowseOverlays.vue.ts

```diff
@@ -62,19 +77,38 @@ export default class BrowseOverlays extends Vue {
     progressCallback?: (progress: IDownloadProgress) => void
   ) {
     for (const url of urls) {
-      const host = (new urlLib.URL(url)).hostname;
+      const host = new urlLib.URL(url).hostname;
       const trustedHosts = ['cdn.streamlabs.com'];
 
       if (!trustedHosts.includes(host)) {
         console.error(`Ignoring widget install from untrusted host: ${host}`);
         return;
       }
 
-      const path = await this.overlaysPersistenceService.downloadOverlay(url, progressCallback);
-      await this.widgetsService.loadWidgetFile(path, this.scenesService.activeSceneId);
+      const path = await this.overlaysPersistenceService.downloadOverlay(
+        url,
+        progressCallback
+      );
+      await this.widgetsService.loadWidgetFile(
+        path,
+        this.scenesService.activeSceneId
+      );
     }
 
     this.navigationService.navigate('Studio');
+
+    this.notificationsService.push({
+      type: ENotificationType.SUCCESS,
+      lifeTime: 8000,
+      showTime: false,
+      message: `Widget Theme installed & activated. Click here to manage your Widget Profiles.`,
+      action: this.jsonrpcService.createRequest(
```

Comment:
Huh this is kind of a weird pattern but I guess it supports pushing notifications with actions over the API.  Anyway, no need to change this, I just haven't seen it before.